e2e/search/search-filters.e2e.ts

```diff
@@ -212,7 +212,17 @@ describe('Search Filters', () => {
         browser.get(TestConfig.adf.url + '/search;q=*');
 
         searchFiltersPage.checkFacetIntervalsByCreatedIsDisplayed()
-            .checkFacetIntervalsByModifiedIsDisplayed();
+            .checkFacetIntervalsByCreatedIsExpanded()
```

Comment:
We should better have another test case for this expand/collapse state... or if you insist to have it here, the test from TestRail must be changed accordingly. @cristinaj what do you recommend?

Reply:
Yes updating the Test case in the Test Rail as well, accordingly. thanks